bin/compare-terraform-to-github.rb

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env ruby
+
+require_relative "../lib/github_collaborators"
+
+terraform_collaborators = GithubCollaborators::TerraformCollaborators.new(
+  base_url: "https://github.com/ministryofjustice/github-collaborators/blob/main/terraform"
+)
+
+repos = GithubCollaborators::Repositories.new(
+  login: "ministryofjustice"
+).current
+
+external_collaborators = GithubCollaborators::OrganizationExternalCollaborators.new(
+  login: "ministryofjustice",
+  base_url: "https://github.com/ministryofjustice/github-collaborators/blob/main/terraform"
+)
+
+# For each repo
+repos.each do |repo|
+  # Get the GitHub collaborators for current repo
+  gc = external_collaborators.for_repository(repo.name)
+  # Get the Terraform collaborators for current repo
+  tc = terraform_collaborators.return_collaborators_from_file("terraform/#{GithubCollaborators.tf_safe(repo.name)}.tf")
+
+  # Edge cases
+  if gc.nil? || tc.nil?
+    next
+  end
+
+  if (gc.length == 0) && (tc.length == 0)
+    next
+  end
+
+  if tc.length == 0
+    next
+  end
```

Comment:
It might be possible to reduce these if statements by using the safe access operator, specifically to remove the nil checks https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/ not sure if needed, but throwing it out there.